### PR TITLE
feat(solidstart): Add kit to CLI

### PR DESCRIPTION
**Pull request by SDaian**
## Type of change

- [x] Documentation change
- [ ] Bug fix

## Summary of change

- [x] Added kit to the CLI

Closes: #910

### CODE REVIEW

File: starters/solidstart-tanstackquery-tailwind-modules/package.json [hunk 1]
```diff
@@ -2,7 +2,13 @@
   "name": "solidstart-tanstackquery-tailwind-modules",
   "version": "0.1.0",
   "private": true,
-  "description": "SolidStart TanStackQuery and TailwindCSS with CSSModules",
+  "description": "SolidStart, TanStackQuery and TailwindCSS with CSSModules",
+  "keywords": [
+    "solidjs",
+    "tan-stack-query",
```
Comment (hdJerry) [inline starters/solidstart-tanstackquery-tailwind-modules/package.json:8]: tanstack is on word

**Review by Megio — COMMENTED** [review]
In the package.json file I see two times this `"@testing-library/jest-dom"`

The rest is good! 🙌🏻

**Review by hdJerry — APPROVED** [review]
You Rock Sir!

File: starters/solidstart-tanstackquery-tailwind-modules/package.json [hunk 2]
```diff
@@ -2,7 +2,13 @@
   "name": "solidstart-tanstackquery-tailwind-modules",
   "version": "0.1.0",
   "private": true,
-  "description": "SolidStart TanStackQuery and TailwindCSS with CSSModules",
+  "description": "SolidStart, TanStackQuery and TailwindCSS with CSSModules",
+  "keywords": [
+    "solidjs",
```
Comment (hdJerry) [inline starters/solidstart-tanstackquery-tailwind-modules/package.json:7]: solidstart

**Review by hdJerry — CHANGES_REQUESTED** [review]
Please make this adjustment above

**Review by hdJerry — APPROVED** [review]
Thanks sir

**Pull request merged**